lectures/input_output.md

```diff
@@ -574,11 +570,12 @@ High ranking sectors within this measure are important buyers of intermediate go
 A demand shock in such sectors will cause a large impact on the whole production network.
 
 The following figure displays the output multipliers for the sectors represented
-in {numref}`us_15sectors`.
+in {ref}`us_15sectors`.
 
 ```{code-cell} ipython3
 :tags: [hide-cell]
 
+A, F = build_coefficient_matrices(Z, X)
```

Comment:
Hi @maanasee,

Many thanks for the great lecture!

I fixed a small bug here as `A` was overwritten by the small `A` defined above, which was also causing errors when drawing the bar chart.

Please kindly check the output to see if they are what you expected :)